view/src/test/java/cmm/apps/esmorga/view/screenshot/eventdetails/EventDetailsScreenshotTest.kt

```diff
@@ -20,15 +22,20 @@ class EventDetailsScreenshotTest : BaseScreenshotTest() {
 
     @Test
     fun eventDetailsView_lightTheme_data_user_event_joined() {
-        snapshotWithState(userJoined = true, isAuthenticate = true)
+        snapshotWithState(buttonTitle = R.string.button_leave_event)
     }
 
     @Test
     fun eventDetailsView_lightTheme_data_user_event_not_joined() {
-        snapshotWithState(userJoined = false, isAuthenticate = true)
+        snapshotWithState(buttonTitle = R.string.button_join_event)
     }
 
-    private fun snapshotWithState(lat: Double? = 0.0, lng: Double? = 2.88, userJoined: Boolean = false, isAuthenticate: Boolean = false) {
+    @Test
+    fun eventDetailsView_lightTheme_data_primary_button_loading_state() {
+        snapshotWithState(buttonLoading = true)
+    }
+
+    private fun snapshotWithState(lat: Double? = 0.0, lng: Double? = 2.88, buttonTitle: Int = R.string.button_login_to_join, buttonLoading: Boolean = false) {
```

Comment:
No test for loading is created, right? then loading parameter should be removed.

Reply:
yes, I added one, the last one 
`@Test
    fun eventDetailsView_lightTheme_data_primary_button_loading_state() {
        snapshotWithState(buttonLoading = true)
    }`